generic/tests/ntttcp.py

```diff
@@ -36,11 +55,13 @@ def run(test, params, env):
     buffers = params.get("buffers").split()
     buf_num = params.get("buf_num", 200000)
     session_num = params.get("session_num")
+    timeout = int(params.get("timeout")) * int(session_num)
+    driver_name = params.get("driver_name", "netkvm")
```

Comment:
suggest move this line into  _verify_vm_driver(), it's only used in this function. And thus, the function also can reduce 1 param driver_name.

Reply:
sorry wyu, my mistake, the params is in run() func not in  _verify_vm_driver().
if the _verify_vm_driver() func only use for this py, i think can move it into run() func, it will only need one param  verify_vm_driver(vm),  just a suggestion~